src/traces/bar/style.js

```diff
@@ -20,8 +20,46 @@ var attributeInsideTextFont = attributes.insidetextfont;
 var attributeOutsideTextFont = attributes.outsidetextfont;
 var helpers = require('./helpers');
 
+function resizeText(gd, gTrace, traceType) {
+    var fullLayout = gd._fullLayout;
+    var minSize = fullLayout['_' + traceType + 'Text_minsize'];
+    if(minSize) {
+        var shouldHide = fullLayout.uniformtext.mode === 'hide';
+
+        var t;
+        switch(traceType) {
+            case 'funnelarea' :
+            case 'pie' :
+            case 'sunburst' :
+            case 'treemap' :
+                t = gTrace.selectAll('g.slicetext').selectAll('text');
+                break;
+            default :
+                t = gTrace.selectAll('g.points').selectAll('g.point').selectAll('text');
+        }
+
+        var isCenter = (
+            traceType === 'pie' ||
+            traceType === 'sunburst'
+        );
+
+        t.each(function(d) {
+            var transform = d.transform;
+
+            transform.scale = minSize / transform.fontSize;
+            d3.select(this).attr('transform', Lib.getTextTransform(transform, isCenter));
+
+            if(shouldHide && transform.hide) {
+                d3.select(this).text(' ');
```

Comment:
Hmm. It's probably best to use `d3.select(this).attr('display', 'none')` to hide text elements. That way we could have something like:

```js
d3.select(this).attr('display', shouldHide && transform.hide ? 'none' : null);
```

similar to 

https://github.com/plotly/plotly.js/blob/b2102a504a480493d9e474e15489c4874d5f6bac/src/components/drawing/index.js#L98-L101

making the hide/show logic independent of `_module.plot`.

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/4420/commits/a27dc29ca87303fd53bf994bfe1203b3c5e96887.

Reply:
Much better. Thanks!!